src/Forum/Forum.router.ts

```diff
@@ -24,7 +25,7 @@ export class ForumRouter extends Router {
       withAuthentication,
       withCollectionExists,
       withCollectionAuthorization,
-      server.handleRequest(this.post)
+      server.handleRequest(this.post.bind(this))
```

Comment:
If you wanna avoid using `bind` here you could also use bound instance methods like the Asset.router or Collection.router do:

```ts
// from
async post(req: AuthRequest) {
}

// to
async post = (req: AuthRequest) => {
}
```

Reply:
I used to prefer using variables with methods, but I switched not too long time ago to use bind. I changed it anyways.